### PR TITLE
Don't select items on mouse right-click

### PR DESCRIPTION
**Pull request by TxHawks**
I think it is a bit counter intuitive that right-clicking an item fires the select function.

I suggest only firing the select function if the `mousedown` event is a right click (`evt.which === 1`).

### CODE REVIEW

File: awesomplete.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ var _ = function (input, o) {
 				li = li.parentNode;
 			}
 
-			if (li) {
+			if (li && evt.which === 1) {  // Only select on left click
```
Comment (LeaVerou) [inline awesomplete.js]: Given that `evt.which` is non-standard, I'd rather do `evt.which !== 2 && evt.which !== 3` or `[2,3].indexOf(evt.which) === -1` or something like that.

**Comment by LeaVerou** [comment]
Thanks! See comment.

File: awesomplete.js (hunk 1 above)
Comment (vlazar) [inline awesomplete.js]: Or https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button ?

Comment (TxHawks) [inline awesomplete.js]: My bad. 

I guess we should probably go with the standard and less convoluted `evt.button === 0` which [seem to be well supported](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button).

**Comment by LeaVerou** [comment]
Yes! Nice find!
Merged, thanks!

**Pull request merged**